docs/tutorials/setting-up-python-pycharm.md

```diff
@@ -1,83 +1,71 @@
 ---
 title: Setting Up a Python Development Environment with PyCharm
 group: software-languages
-author: "@antonvh"
+author: "@antonvh, @fisherds"
```

Comment:
This needs to be written as a yaml list.

    author: [ "@antonvh", "@fisherds" ]